<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SyvDaDashboardToolbox</class>
 <widget class="QWidget" name="C_SyvDaDashboardToolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="stw::opensyde_gui_elements::C_OgeTawToolBox" name="pc_TabWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <widget class="QWidget" name="pc_TabWidgets">
      <attribute name="title">
       <string>Widgets</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="stw::opensyde_gui_elements::C_OgeSmoothScrollArea" name="pc_ScrollAreaWidgets">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="stw::opensyde_gui_elements::C_OgeWiOnlyBackground" name="pc_ScrollAreaWidgetsWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>988</width>
            <height>675</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>14</number>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="pc_VerticalLayout1"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pc_TabFunctionBlocks">
      <attribute name="title">
       <string>Function Blocks</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="stw::opensyde_gui_elements::C_OgeSmoothScrollArea" name="pc_ScrollAreaFunctionBlocks">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="stw::opensyde_gui_elements::C_OgeWiOnlyBackground" name="pc_ScrollAreaFunctionBlocksWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>988</width>
            <height>675</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="leftMargin">
            <number>14</number>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="pc_VerticalLayout3">
             <property name="topMargin">
              <number>0</number>
             </property>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pc_TabDrawing">
      <attribute name="title">
       <string>Drawing</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="stw::opensyde_gui_elements::C_OgeSmoothScrollArea" name="pc_ScrollAreaDrawing">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="stw::opensyde_gui_elements::C_OgeWiOnlyBackground" name="pc_ScrollAreaDrawingWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>988</width>
            <height>675</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="leftMargin">
            <number>14</number>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="pc_VerticalLayout2" stretch="0,0,1">
             <item>
              <widget class="stw::opensyde_gui_elements::C_OgeLabToolboxHeadingGroup" name="pc_LabelFreeElements">
               <property name="text">
                <string>Free Elements</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="stw::opensyde_gui::C_SdTopologyListWidget" name="pc_ListWidgetDrawing">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="pc_VerticalSpacer2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeWiOnlyBackground</class>
   <extends>QWidget</extends>
   <header>C_OgeWiOnlyBackground.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui::C_SdTopologyListWidget</class>
   <extends>QListWidget</extends>
   <header>C_SdTopologyListWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeTawToolBox</class>
   <extends>QTabWidget</extends>
   <header>C_OgeTawToolBox.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeLabToolboxHeadingGroup</class>
   <extends>QLabel</extends>
   <header>C_OgeLabToolboxHeadingGroup.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeSmoothScrollArea</class>
   <extends>QScrollArea</extends>
   <header>C_OgeSmoothScrollArea.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
